streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -139,8 +140,9 @@ public StreamTask(final TaskId id,
         initializeStateStores();
         stateMgr.registerGlobalStateStores(topology.globalStateStores());
         if (eosEnabled) {
-            producer.initTransactions();
-            producer.beginTransaction();
+            this.producer.initTransactions();
```

Comment:
We don't -- but it help Intellij to highlight all LOC where the variable is used if you click on it -- otherwise, constructor access is missed ;)